src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -351,7 +350,7 @@
     <value>Compare symbols correctly</value>
   </data>
   <data name="ConfigureGeneratedCodeAnalysisDescription" xml:space="preserve">
```

Comment:
We should delete this resource string and update the analyzer to not pass in a description. It has no useful information over the title/message.

Reply:
I prefer to keep both resources so it's clear what can be updated

Reply:
How is it valuable over the diagnostic message?

Reply:
You guys are too fast, I was adding my notes xD

We have a LOT of resources where the title, message and description are the same. I am fine with both approach but going on Sam's direction would make our analyzers simpler. Otherwise we will have to have special cases that might lead to false negative.

Reply:
Hum actually nevermind I got your suggestion wrong. Yes +1 to remove the description when it's the same at the message.